wiki/Manual.md

```diff
@@ -4,44 +4,46 @@ The ccxt library is a collection of available crypto *exchanges* or exchange cla
 
 The structure of the library can be outlined as follows:
 
-```text
-                                 User
-    +-------------------------------------------------------------+
-    |                            CCXT                             |
-    +------------------------------+------------------------------+
-    |            Public            |           Private            |
-    +=============================================================+
-    │                              .                              |
-    │                    The Unified CCXT API                     |
-    │                              .                              |
-    |       loadMarkets            .           fetchBalance       |
-    |       fetchMarkets           .            createOrder       |
-    |       fetchCurrencies        .            cancelOrder       |
-    |       fetchTicker            .             fetchOrder       |
-    |       fetchTickers           .            fetchOrders       |
-    |       fetchOrderBook         .        fetchOpenOrders       |
-    |       fetchOHLCV             .      fetchClosedOrders       |
-    |       fetchStatus            .          fetchMyTrades       |
-    |       fetchTrades            .                deposit       |
-    |                              .               withdraw       |
-    │                              .                              |
-    +=============================================================+
-    │                              .                              |
-    |                     Custom Exchange API                     |
-    |         (Derived Classes And Their Implicit Methods)        |
-    │                              .                              |
-    |       publicGet...           .          privateGet...       |
-    |       publicPost...          .         privatePost...       |
-    |                              .          privatePut...       |
-    |                              .       privateDelete...       |
-    |                              .                   sign       |
-    │                              .                              |
-    +=============================================================+
-    │                              .                              |
-    |                      Base Exchange Class                    |
-    │                              .                              |
-    +=============================================================+
-```
+<pre>
```

Comment:
@newillusion noticing that this text is not centered 
![image](https://github.com/ccxt/ccxt/assets/43336371/f481d115-61dd-48b0-a61e-b7aca94dfef3)

Reply:
also the table itself is not centered, can you do it?